### ISSUE TITLE
Experiment details

### ISSUE THREAD

**Issue by KerwenX** [yccyenchicheng/SDFusion#13]
Hello @yccyenchicheng,

I apologize for the inconvenience and hope this message finds you well. I find your research to be highly intriguing and I have some inquiries regarding the experimental details.

Specifically, I would like to inquire about the device(s) used when training the `Sdfusion-mm2shape`. Additionally, I am curious to know how long it took to complete the training process.

Furthermore, I would like to kindly request if you could consider releasing the training code for `Sdfusion-mm2shape` as soon as possible.

Thank you very much for your time and consideration. I appreciate any assistance you can provide!

Best regards

**Comment by yhhxt** [comment]
I have a similar question. I am training a VQ-VAE using DDP framework with 4 NVIDIA GeForce RTX 3090 GPUs. I am using the entire Shapenet dataset, which includes all classes, but the training time is as long as 40,000 hours. I was wondering what dataset you used and how many epochs you ran the model for when you trained a similar model. Also, how long did the training take?Thanks.
![b23daf2e62704e0f6f80a18f5696eda](https://user-images.githubusercontent.com/55875788/236200373-8e99bf90-f526-47e1-aabe-586f3141a769.png)

**Comment by KerwenX** [comment]
Thank you for getting back to me, and I apologize for the delay in my response. However, I have not yet started training on this project, so I am unable to provide an answer to your question at this time.

**Comment by KerwenX** [comment]
> I have a similar question. I am training a VQ-VAE using DDP framework with 4 NVIDIA GeForce RTX 3090 GPUs. I am using the entire Shapenet dataset, which includes all classes, but the training time is as long as 40,000 hours. I was wondering what dataset you used and how many epochs you ran the model for when you trained a similar model. Also, how long did the training take?Thanks. ![b23daf2e62704e0f6f80a18f5696eda](https://user-images.githubusercontent.com/55875788/236200373-8e99bf90-f526-47e1-aabe-586f3141a769.png)

Thank you for getting back to me, and I apologize for the delay in my response. However, I have not yet started training on this project, so I am unable to provide an answer to your question at this time.

**Comment by yccyenchicheng** [comment]
You can monitor through the tensorboard and stops the training if the loss converges or the reconstruction looks decent for you.

It takes around 7-10 days to train the AE, and 10-14 days to train the diffusion models on one GPU. It will converge faster with multiple GPUs with DDP.

**Comment by yhhxt** [comment]
Thanks a lot!
